integrations/mongodb.md

```diff
@@ -16,17 +16,64 @@ logo: /logos/mongodb.png
 toc: true
```

Comment:
```suggestion
toc: true
version: Haystack 2.0
```
Also, let's update the links in the frontmatter. They should point to the haystack-core-integration code